ProcessMaker/Models/ProcessRequest.php

```diff
@@ -552,6 +553,7 @@ public function logError(Throwable $exception, FlowElementInterface $element = n
         \Log::error($exception);
```

Comment:
If this `\Log::error()` is not required or was used for debugging, please remove it.